src/Templates/src/templates/maui-mobile/AppShell.xaml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Shell
+    x:Class="MauiApp._1.AppShell"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:MauiApp._1"
+    FlyoutBackgroundColor="{AppThemeBinding Dark={StaticResource DarkColor},
+                                            Light={StaticResource LightColor}}"
+    Shell.FlyoutBehavior="Disabled">
+    <Shell.Resources>
+        <ResourceDictionary>
+            <Style x:Key="BaseStyle" TargetType="Element">
+                <Setter Property="Shell.BackgroundColor" Value="{AppThemeBinding Dark={StaticResource DarkColor}, Light={StaticResource LightColor}}" />
+                <Setter Property="Shell.ForegroundColor" Value="{AppThemeBinding Dark={StaticResource LightColor}, Light={StaticResource DarkColor}}" />
+                <Setter Property="Shell.TitleColor" Value="{AppThemeBinding Dark={StaticResource LightColor}, Light={StaticResource DarkColor}}" />
+                <Setter Property="Shell.DisabledColor" Value="#B4FFFFFF" />
+                <Setter Property="Shell.UnselectedColor" Value="{AppThemeBinding Dark=#95FFFFFF, Light=#95000000}" />
+                <Setter Property="Shell.TabBarBackgroundColor" Value="{AppThemeBinding Dark={StaticResource DarkColor}, Light={StaticResource LightColor}}" />
+                <Setter Property="Shell.TabBarForegroundColor" Value="{AppThemeBinding Dark={StaticResource LightColor}, Light={StaticResource DarkColor}}" />
+                <Setter Property="Shell.TabBarUnselectedColor" Value="{AppThemeBinding Dark=#95FFFFFF, Light=#95000000}" />
+                <Setter Property="Shell.TabBarTitleColor" Value="{AppThemeBinding Dark={StaticResource LightColor}, Light={StaticResource DarkColor}}" />
+            </Style>
+            <Style BasedOn="{StaticResource BaseStyle}" TargetType="TabBar" />
+            <Style BasedOn="{StaticResource BaseStyle}" TargetType="FlyoutItem" />
+        </ResourceDictionary>
+    </Shell.Resources>
+
+
+
+    <ShellContent
+        Title="Hello, World!"
+        ContentTemplate="{DataTemplate local:MainPage}"
+        Route="MainPage" />
```

Comment:
Hey, I wanted to bring up. What is the impact of startup time for these changes to the template?

I'd be curious of the average of 10 runs on a device using the script here:

https://github.com/jonathanpeppers/maui-profiling/blob/main/scripts/profile.ps1

Comparing a regular release build would be fine.

We should do this if it is better, but maybe we also want some buy-in from the MAUI team to improve any startup cost when adding Shell by default?

Reply:
Ideally it is no impact... as that should be the goal of shell for sure.

Reply:
Non-Shell
Average(ms): 2682.8
Std Err(ms): 18.5159390796146
Std Dev(ms): 58.552540508504

Shell:
Average(ms): 3489.2
Std Err(ms): 45.70285670624
Std Dev(ms): 144.52512276802

Reply:
I saw some things in `dotnet trace` like this line taking up 20ms:

https://github.com/dotnet/maui/blob/303e3ef5e084419ed94a0b7a93171479711c92fa/src/Controls/src/Core/Shell/ShellUriHandler.cs#L243

I bet your numbers would be a bit better on a device. Is it also a Release build?

Reply:
running on device now

Reply:
Added details